tests/test_enterprise/api/test_views.py

```diff
@@ -8495,6 +8495,20 @@ def test_successful_update_group(self):
         assert response.json().get('enterprise_customer') == str(new_uuid)
         assert len(EnterpriseGroup.objects.filter(enterprise_customer=str(new_uuid))) == 1
 
+    def test_successful_update_group_name(self):
+        """
+        Test patching an existing group record display name
+        """
+        # url: 'http://testserver/enterprise/api/v1/enterprise_group/<group uuid>'
```

Comment:
nit: can you update the url to include a forward slash at the end of the url since that's required to make the patch request? `http://testserver/enterprise/api/v1/enterprise_group/<group uuid>/`